vulnerabilities/importers/github.py

```diff
@@ -20,315 +20,528 @@
 #  VulnerableCode is a free software  from nexB Inc. and others.
 #  Visit https://github.com/nexB/vulnerablecode/ for support and download.
 
-import asyncio
-import dataclasses
+import logging
 import os
+from datetime import datetime
+from typing import Iterable
 from typing import List
 from typing import Mapping
 from typing import Optional
-from typing import Set
 from typing import Tuple
 
 import requests
 from dateutil import parser as dateparser
+from django.db.models.query import QuerySet
 from packageurl import PackageURL
-from univers.version_specifier import VersionSpecifier
-from univers.versions import version_class_by_package_type
+from univers.version_range import VersionRange
+from univers.version_range import build_range_from_github_advisory_constraint
```

Comment:
Should `build_range_from_github_advisory_constraint` be  a class factory method of https://github.com/nexB/univers/blob/bc8e33fd923c5ec1dbe16528d3e207015f0460e2/src/univers/version_range.py#L714 

I would move https://github.com/nexB/univers/blob/bc8e33fd923c5ec1dbe16528d3e207015f0460e2/src/univers/version_range.py#L902 and https://github.com/nexB/univers/blob/bc8e33fd923c5ec1dbe16528d3e207015f0460e2/src/univers/version_range.py#L912 to the GitHubVersionRange class.

Reply:
Note that this can be done separately and is not in the way of merging this.